cli/src/cmd/forge/build/core.rs

```diff
@@ -153,7 +153,7 @@ impl<'a> From<&'a CoreBuildArgs> for Figment {
         remappings
             .extend(figment.extract_inner::<Vec<Remapping>>("remappings").unwrap_or_default());
         remappings.sort_by(|a, b| a.name.cmp(&b.name));
-        remappings.dedup_by(|a, b| a.name.eq(&b.name));
+        remappings.dedup_by(|a, b| (&a.context, &a.name).eq(&(&b.context, &b.name)));
```

Comment:
this looks correct we don't want duplicate remappings with same context and name.

would be great to have a test for this

Reply:
Just to double check here: We don't wanna duplicate remappings that have _both_ the same context and name. Them having same name but different contexts should be OK